### PR TITLE
[MAINTENANCE] Loosen raumel pin for CVE-2019-20478

### LINKED ISSUE #10163: Allow ruamel.yaml package version > 0.18 to avoid CVE-2019-20478
[This issue](https://nvd.nist.gov/vuln/detail/CVE-2019-20478) appear in all the versions for the ruamel.yaml < 0.18.
this violations is fixed for the version > 0.18.

Would be great if greatexpectation tool is adjusted to use ruamel.yaml package version > 0.18 to get rid of CVE error!

Thanks!

### PR DESCRIPTION
**Pull request by Kilo59**
Remove raumel pin to allow for installation of `0.18` or greater which resolves a CVE.
https://nvd.nist.gov/vuln/detail/CVE-2019-20478

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -24,7 +24,7 @@ pyparsing>=2.4
 python-dateutil>=2.8.1
 pytz>=2021.3
 requests>=2.20
-ruamel.yaml>=0.16,<0.18  # See https://github.com/great-expectations/great_expectations/issues/4152
```
Comment (tyler-hoffman) [inline requirements.txt:27]: Following this link, it shows that `context.test_yaml_config(yaml.dump(datasource_config))` is deprecated, and we still have a number of those calls. Should we get those fixed?

File: requirements.txt [hunk 2]
```diff
@@ -24,7 +24,7 @@ pyparsing>=2.4
 python-dateutil>=2.8.1
 pytz>=2021.3
 requests>=2.20
-ruamel.yaml>=0.16
+ruamel.yaml>=0.18 # TODO: revert to 0.16 before merge assuming tests pass
```
Comment (Kilo59) [inline requirements.txt]:
```suggestion
ruamel.yaml>=0.16
```
Temporarily force installing `0.18` just to ensure no tests break.

File: requirements.txt (hunk 1 above)
Comment (Kilo59) [reply requirements.txt:27]: Yes, I had assumed those were already removed in 1.0.

Comment (tyler-hoffman) [reply requirements.txt:27]: Oh ya know what? If I filter out versioned docs, it shows we don't call it. You should be good to go with this 😄

**Review by tyler-hoffman — APPROVED** [review]

**Pull request merged**